Update _docHome.md (#9)
pingcap/docs-staging: markdown-pages/en/tidb/_docHome.md

<DocHomeContainer title="PingCAP Archived Documentation" subTitle="Access the archived documentation of the TiDB database in this website" archive>

<DocHomeSection label="TiDB" anchor="tidb" id="tidb">
  
This website provides archived TiDB documentation for the following legacy TiDB versions:

> - TiDB Long Term Support (LTS) versions that are after End of Life (EOL) dates

Reviewer: To keep it consistent with that in https://docs.pingcap.com/tidb/stable/versioning
```suggestion
- TiDB Long-Term Support (LTS) versions that are after End of Life (EOL) dates
```

> - TiDB Development Milestone Release (DMR) versions that are not rencently published

Reviewer: ```suggestion
- TiDB Development Milestone Release (DMR) versions that are not recently released
```

PingCAP encourages you to use [the latest LTS version](https://docs.pingcap.com/tidb/stable/) of the TiDB database. Archived documentation no longer receives updates.

> | Release    | Documentation                                                  |

Author: ```suggestion
| Release    | Archived documentation                                                  |
```

| ---------- | -------------------------------------------------------------- |
| v6.2 (DMR) | [TiDB v6.2 documentation](https://docs.pingcap.com/tidb/v6.2/) |
| v6.0 (DMR) | [TiDB v6.0 documentation](https://docs.pingcap.com/tidb/v6.0/) |
| v3.1       | [TiDB v3.1 documentation](https://docs.pingcap.com/tidb/v3.1/) |
| v2.1       | [TiDB v2.1 documentation](https://docs.pingcap.com/tidb/v2.1/) |

> [Learn more about TiDB Release Support Policy](https://www.pingcap.com/tidb-release-support-policy/?from=en)

Reviewer: ```suggestion
Learn more about [TiDB Release Support Policy](https://www.pingcap.com/tidb-release-support-policy/?from=en).
```
